### PR TITLE
[WnD] add docker file for wnd

### PR DESCRIPTION
**Pull request by Peach-He**

### CODE REVIEW

**Comment by Jian-Zhang** [comment]
So the conclusion is the v0.1 docker does not work for WnD? @Peach-He  @zigzagcai ?

**Comment by Peach-He** [comment]
> So the conclusion is the v0.1 docker does not work for WnD? @Peach-He @zigzagcai ?

Original v0.1 docker doesn't work for WnD with horovodrun, This new dockerfile works for WnD. Since only WnD and DIEN use tensorflow env in v0.1, if this dockerfile works for DIEN as well, I think we can replace the old dockerfile.

**Comment by zigzagcai** [comment]
> So the conclusion is the v0.1 docker does not work for WnD? @Peach-He @zigzagcai ?

original v0.1 TF docker will meet some issue with mpirun backend, so that for some workloads like WnD, the backend can be switched to horovodrun.

**Comment by xuechendi** [comment]
@Peach-He , I am verifying this PR, the spark version is  3.0, can we use 3.2.1 to align with pytorch version?

**Comment by xuechendi** [comment]
> @Peach-He , I am verifying this PR, the spark version is 3.0, can we use 3.2.1 to align with pytorch version?

RecDP is better working on 3.2 version, since there are API changes since 3.1 and 3.0 lacks some skew join support

**Comment by xuechendi** [comment]
@Peach-He , I tested to use this docker running dien, and it runs perfectly. Please update spark version and then we can replace current DockerFileTensorflow

**Comment by Jian-Zhang** [comment]
Pls. add jenkins test label when you update it.

**Comment by Peach-He** [comment]
> Pls. add jenkins test label when you update it.

looks like I don't have permission to add jenkins test label, can you help to add the label?

**Comment by Peach-He** [comment]
I am going to replace DockerFileTensorflow with this dockerfile after upgrading spark version, since this dockerfile works well with DIEN and WnD.

**Comment by Jian-Zhang** [comment]
was spark version updated?

**Comment by xuechendi** [comment]
merged, I checked CI/CD, this docker file is working well and spark is now on version 3.2.1
